client/src/features/User/components/EmptyOrderList/EmptyOrderList.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { Paper, Typography, Button } from '@mui/material';
+
+const emptyOrderListSX = {
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+  flexDirection: 'column',
+  borderRadius: '20px',
+};
+
+const emptyOrderListMessage = {
+  pt: { xs: '20px', tablet: '40px', desktop: '60px' },
```

Comment:
i see the whole component is useless code with no real functionality,
instead you could approach this feature in a very simple way.

in the userOrders component,  if(orders && orders.length === 0)
just render a typogrphy component variant h2 with simple text and we are done!!  well may be you might add a button to navigate back to catalogue page and that it is more than enough

Reply:
in this case, the UI looks prettier, and the code remains simple and readable.  I don't see any problem here

Reply:
YAGNI and KISS violation.